src/locale/DE/modal.js

```diff
@@ -0,0 +1,215 @@
+import { createState } from '../../utils';
+
+export function getModalType() {
+    return 'INST';
+}
+
+export default function getModalContent(options, state, trackModalEvent) {
+    const type = getModalType();
+
+    const [carouselState, setCarouselState] = createState({
+        activeItem: 0
+    });
+
+    const getElements = iframe => {
+        const financeTermsTable = iframe.contentDocument.getElementById('terms-table');
+        const loader = iframe.contentDocument.getElementById('loading-image');
+        const carouselWrapper = iframe.contentDocument.getElementById('carousel-background');
+        const carouselSlider = iframe.contentDocument.getElementById('carousel-inner');
+        const carouselItems = iframe.contentDocument.getElementsByClassName('carousel-item');
+        const carouselIndicators = iframe.contentDocument.getElementsByClassName('carousel-bullet');
+        const prevButton = iframe.contentDocument.getElementById('carousel-arrow-prev');
+        const nextButton = iframe.contentDocument.getElementById('carousel-arrow-next');
+        const amountInput = iframe.contentDocument.getElementById('amount-input');
+        const calculatorInstructions = iframe.contentDocument.getElementById('calculator-instructions');
+        const calculateButton = iframe.contentDocument.getElementById('calculate-button');
+
+        return {
+            financeTermsTable,
+            loader,
+            carouselWrapper,
+            carouselSlider,
+            carouselItems,
+            carouselIndicators,
+            prevButton,
+            nextButton,
+            amountInput,
+            calculatorInstructions,
+            calculateButton
+        };
+    };
+
+    function fetchTerms(amount) {
+        const { loader, financeTermsTable, calculatorInstructions, amountInput } = state.contentElements;
+        // const convertedAmount = +amount;
+        // if (!numberIsNaN(convertedAmount)) {
+        //     // eslint-disable-next-line no-param-reassign
+        //     state.contentElements.amountInput.value = convertedAmount.toFixed(2);
+        // }
+
+        loader.style.setProperty('opacity', 1);
+        financeTermsTable.style.setProperty('opacity', 0.4);
+        financeTermsTable.style.setProperty('min-height', '100px');
+
+        return getTerms({ ...options, amount }).then(terms => {
+            loader.style.setProperty('opacity', 0);
+            financeTermsTable.style.setProperty('opacity', 1);
+            financeTermsTable.style.setProperty('min-height', 'unset');
+            if (amount) {
+                financeTermsTable.innerHTML = renderTermsTable(terms);
+            }
+
+            amountInput.value = terms.formattedAmount;
+            calculatorInstructions.innerText = `Geben Sie einen Betrag zwischen ${terms.formattedMinAmount}€ und ${terms.formattedMaxAmount}€ ein.`;
+        });
+    }
+
+    const fixAmount = amount => amount.replace(/\./, '').replace(/,/, '.');
```

Comment:
Should this be named something more like `localizeAmount` since fix can mean numerous things when referencing a number??